tests/nightly/test_large_array.py

```diff
@@ -689,6 +693,7 @@ def check_pick():
         res = mx.nd.pick(a, b)
         assert res.shape == b.shape
 
+    @unittest.skip("Memory doesn't free up after stacked execution with other ops, tracked at https://github.com/apache/incubator-mxnet/issues/17411")
```

Comment:
this line is too long. please break it to meet PEP8 style

Reply:
aah ... got it ! Will do